packages/core/src/renderers/Control.ts

```diff
@@ -1,25 +1,65 @@
-import { update } from '../actions';
 import { Renderer, RendererProps } from './renderer';
 
-export interface ControlClassNames {
-  input: string;
-  label: string;
-  wrapper: string;
-  description: string;
-}
-
+/**
+ * Props of a renderer.
+ */
 export interface ControlProps extends RendererProps {
+
+  /**
+   * The data to be rendered.
+   */
   data: any;
+
+  /**
+   * The absolute path to the value being rendered.
+   * A path is a sequence of property names separated by dots,
+   * e.g. for accessing the value of b in the object
+   * { foo: { a: { b: 42 } } }, one would use foo.a.b.
+   */
   path: string;
+
+  /**
+   * Path of the parent renderer, if any.
+   */
   parentPath?: string;
-  classNames: ControlClassNames;
+
+  /**
+   * An unique ID that can be used to identify the rendered form.
```

Comment:
that is the id of a single control, so maybe `element` instead of `form`

Reply:
I'm fine with element, updated.